java-checks/src/main/java/org/sonar/java/checks/tests/AssertionArgumentOrderCheck.java

```diff
@@ -77,11 +88,14 @@ protected MethodMatchers getMethodInvocationMatchers() {
 
   @Override
   protected void onMethodInvocationFound(MethodInvocationTree mit) {
-    if (mit.symbol().owner().type().is(ORG_JUNIT5_ASSERTIONS)) {
+    Type ownerType = mit.symbol().owner().type();
+    if (ownerType.is(ORG_JUNIT5_ASSERTIONS)) {
       checkArguments(mit.arguments().get(0), mit.arguments().get(1), "expected value, actual value");
```

Comment:
This message could be extracted to string constants, cause it's used in few places.